docs/en/13-operation/01-pkg-install.md

```diff
@@ -1,152 +1,63 @@
--rw-rw-r--  1 ubuntu ubuntu 44852544 Feb 22 09:31 TDengine-enterprise-server-2.4.0.7-Linux-x64.tar.gz
-
-$ cd TDengine-enterprise-server-2.4.0.7/
-
- $ ll
-total 40784
-drwxrwxr-x  4 ubuntu ubuntu     4096 Feb 22 09:30 ./
-drwxrwxr-x  3 ubuntu ubuntu     4096 Feb 22 09:31 ../
-drwxrwxr-x  2 ubuntu ubuntu     4096 Feb 22 09:30 driver/
-drwxrwxr-x 10 ubuntu ubuntu     4096 Feb 22 09:30 examples/
--rwxrwxr-x  1 ubuntu ubuntu    33294 Feb 22 09:30 install.sh*
--rw-rw-r--  1 ubuntu ubuntu 41704288 Feb 22 09:30 taos.tar.gz
-
-$ sudo ./install.sh
-
-Start to update TDengine...
-Created symlink /etc/systemd/system/multi-user.target.wants/taosd.service → /etc/systemd/system/taosd.service.
-Nginx for TDengine is updated successfully!
-
-To configure TDengine : edit /etc/taos/taos.cfg
-To configure Taos Adapter (if has) : edit /etc/taos/taosadapter.toml
-To start TDengine     : sudo systemctl start taosd
-To access TDengine    : use taos -h ubuntu-1804 in shell OR from http://127.0.0.1:6060
-
-TDengine is updated successfully!
-Install taoskeeper as a standalone service
-taoskeeper is installed, enable it by `systemctl enable taoskeeper`
+$ ll
+total 28
+drwxr-xr-x  7 root root 4096 Feb 22 09:34 ./
+drwxr-xr-x 12 root root 4096 Feb 22 09:34 ../
+drwxr-xr-x  2 root root 4096 Feb 22 09:34 bin/
+drwxr-xr-x  2 root root 4096 Feb 22 09:34 cfg/
+lrwxrwxrwx  1 root root   13 Feb 22 09:34 data -> /var/lib/taos/
+drwxr-xr-x  2 root root 4096 Feb 22 09:34 driver/
+drwxr-xr-x 10 root root 4096 Feb 22 09:34 examples/
+drwxr-xr-x  2 root root 4096 Feb 22 09:34 include/
+lrwxrwxrwx  1 root root   13 Feb 22 09:34 log -> /var/log/taos/
 ```
 
-:::info
-Users will be prompted to enter some configuration information when install.sh is executing. The interactive mode can be disabled by executing `./install.sh -e no`. `./install.sh -h` can show all parameters with detailed explanation.
-
-:::
-
-</TabItem>
-</Tabs>
-
-:::note
-When installing on the first node in the cluster, at the "Enter FQDN:" prompt, nothing needs to be provided. When installing on subsequent nodes, at the "Enter FQDN:" prompt, you must enter the end point of the first dnode in the cluster if it is already up. You can also just ignore it and configure it later after installation is finished.
-
-:::
+- Configuration directory, data directory, and log directory are created automatically if they don't exist
+- The default configuration file is located at /etc/taos/taos.cfg, which is a copy of /usr/local/taos/cfg/taos.cfg
+- The default data directory is /var/lib/taos, which is a soft link to /usr/local/taos/data
+- The default log directory is /var/log/taos, which is a soft link to /usr/local/taos/log
+- The executables at /usr/local/taos/bin are linked to /usr/bin
+- The DLL files at /usr/local/taos/driver are linked to /usr/lib
+- The header files at /usr/local/taos/include are linked to /usr/include
 
 ## Uninstall
 
 <Tabs>
+<TabItem label="apt-get uninstall" value="aptremove">
+
+TBD
```

Comment:
This part has been enhanced, actual "apt-get remove" has been added. You need to merge the change on 3.0 branch to your local branch, or resolve conflicting when merging.